### PR TITLE
Instadapp Fluid: add borrowed amounts

### PR DESCRIPTION
**Pull request by bergben**
Adds borrowed amount stats to Instadapp Fluid. 

Test run output: 
image

Confirms that values are in line with our stats at https://fluid.instadapp.io/stats/1/liquidity

### CODE REVIEW

File: projects/fluid/index.js [hunk 1]
```diff
@@ -13,5 +17,28 @@ module.exports={
...
+            const borrowTokenAmounts = (
+                await api.call({
+                target: fluidLiquidityResolver,
+                abi: fluidLiquidityResolverAbi,
+                params: [tokens],
+                })
+            ).map((x, i) => ({ token: tokens[i], borrow: x.totalBorrow}));
+
+            for await (const borrowTokenAmount of borrowTokenAmounts){
+                await api.add(
```
Comment (bergben) [inline projects/fluid/index.js:36]: LMK if there is a better way than looping through here to add multiple tokens with amounts at once

**Comment by llamatester** [comment]
The adapter at projects/fluid exports TVL: 
        
 
 
    ethereum-borrowed         237.17 M
    borrowed                  237.17 M
    ethereum                  185.07 M
    
    total                    185.07 M

**Comment by llamatester** [comment]
The adapter at projects/fluid exports TVL: 
        
 
 
    ethereum-borrowed         309.34 M
    borrowed                  309.34 M
    ethereum                  238.66 M
    
    total                    238.66 M

**Comment by g1nt0ki** [comment]
@bergben thanks for the PR, sorry for the delay

**Comment by bergben** [comment]
Hey @g1nt0ki thanks for refactoring and merging! Wanted to ask if there is a way to run a historical data sync if such logic is added? to have borrowed amounts tracked for the past 2 months as well? 

fyi I will also work on getting other data related to Fluid updated like fees revenue etc.

**Comment by bergben** [comment]
> for some reason, the refill script stops on April 16th, cant fill before that :(

oh, that is because of the contract address of `const fluidLiquidityResolver = "0x645C84DeA082328e456892D2E68d434b61AD7dBF";`. This one was not deployed before April 16th... Before that, we'd have to use the address `0x741c2Cd25f053a55fd94afF1afAEf146523E1249`, which tracks back to Feb 16th. everything logic related itself could stay the exact same! Would that be possible?

**Comment by g1nt0ki** [comment]
ah, you are right, it is now refilled till feb 16th, should reflect on the UI in few hours.

**Comment by bergben** [comment]
Hey @g1nt0ki the back sync still is not reflected on UI, just to check in is this expected?

**Comment by g1nt0ki** [comment]
think it is a cache issue, let me try to fix it

**Pull request merged**